assets/src/dashboard/icons/arrowAlphaDescending.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2 11l6.5-6.5m0 0L15 11M8.5 4.5v17" stroke="currentColor" stroke-width="2"/>
+    <path d="M18.911 21.656l4.5-6h-4.308v-1.248h5.832v1.344l-4.464 6h4.512V23h-6.072v-1.344zM21.742 3.408h1.5L26.494 12h-1.452l-.792-2.208h-3.504L19.954 12h-1.452l3.24-8.592zm2.076 5.16l-.96-2.628-.324-.984h-.072l-.324.984-.96 2.628h2.64z" fill="currentColor"/>
+</svg>
```

Comment:
And this is ascending:
![image](https://user-images.githubusercontent.com/40646372/83277955-a8a0f880-a187-11ea-8b96-1874694c238f.png)

Reply:
thanks :) i just pushed up a change for this.